src/main/java/frc/robot/RobotContainer.java

```diff
@@ -17,20 +16,21 @@
  * subsystems, commands, and button mappings) should be declared here.
  */
 public class RobotContainer {
-  // The robot's subsystems and commands are defined here...
-  private final ExampleSubsystem m_exampleSubsystem = new ExampleSubsystem();
-
-  private final ExampleCommand m_autoCommand = new ExampleCommand(m_exampleSubsystem);
-
+  // driveTrain subsystem 
```

Comment:
Follow other comments format and use first capital letter as the comment is a sentence. Do this for all comments, I won't open a review for each of them.

```suggestion
  // DriveTrain subsystem 
```